### PR TITLE
Add FAQs on how to know if a job fails

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/data-retention.rst [hunk 1]
```diff
@@ -78,3 +78,11 @@ In v1.5 and later of the iOS and Android apps, messages and files are deleted fr
 2. When the user puts the app into the background.
 
 In v1.4 and earlier of the mobile apps, messages and files are not cleared from local storage when the data retention policy is enabled.
+
+How do I know if a data retention job fails?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost provides the status of each data retention job in **System Console** > **Compliance** > **Data Retention Policy**. Here, you can see if the job succeeded or failed, including XXX.
```
Comment (jasonblais) [inline source/administration/data-retention.rst]: Katie: wondering if you know what's included in the status table -- # of messages deleted, perhaps?

File: source/deployment/elasticsearch.rst [hunk 2]
```diff
@@ -90,3 +90,11 @@ There's a few possible reasons why it might be slow:
  - The Elasticsearch cluster is performance limited (i.e. the machines are not powerful enough).
 
  - The 25,000 messages are spread out over a long time window, and the ``BulkIndexingTimeWindowSeconds`` configuration value is too low for efficient indexing of such a "sparse" database. Optimally the value of that config should be set such that the median number of posts falling within any period of that time in the database is around 700 to 800. The default value is 1 hour, so if you are doing a lot less than 800 posts an hour on average, then the indexing will be much slower in terms of "posts per unit time". This can be sped up by increasing that time window.
+
+How do I know if an elasticsearch job fails?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost provides the status of each elasticsearch indexing job in **System Console** > **Environment** > **Elasticsearch** (or **System Console > Advanced > Elasticsearch** in versions 5.11 and earlier). Here, you can see if the job succeeded or failed, including the XXX.
```
Comment (jasonblais) [inline source/deployment/elasticsearch.rst]: Same here about elasticsearch job updates

Comment (wiersgallak) [reply source/deployment/elasticsearch.rst]: Please add : "including the details of the error."

File: source/administration/data-retention.rst (hunk 1 above)
Comment (wiersgallak) [reply source/administration/data-retention.rst]: Please add : "including the details of the error."  The grid includes Status, Finish Time, Run Time and Details.  Details is only populated when there is an error.

**Comment by jasonblais** [comment]
@wiersgallak Thanks! Updated

File: source/deployment/sso-ldap.md [hunk 3]
```diff
@@ -180,3 +180,9 @@ You can find an explanation of each of the configuration settings [here](https:/
 Yes it can, but make sure that:
  - permissions are correctly configured on the service account you are using
  - each user object is a direct member of the security group
+
+#### How do I know if an AD/LDAP sync job fails?
+
+Mattermost provides the status of each AD/LDAP sync job in **System Console** > **Authentication** > **AD/LDAP**. Here, you can see if the job succeeded or failed, including the number of users updated.
```
Comment (wiersgallak) [inline source/deployment/sso-ldap.md]: Propose: "Here you can see the number of users updated and if the job succeeded or failed."

Comment (jasonblais) [reply source/deployment/sso-ldap.md]: @wiersgallak Thanks! Updated

File: source/deployment/sso-ldap.md [hunk 4]
```diff
@@ -180,3 +180,9 @@ You can find an explanation of each of the configuration settings [here](https:/
 Yes it can, but make sure that:
  - permissions are correctly configured on the service account you are using
  - each user object is a direct member of the security group
+
+#### How do I know if an AD/LDAP sync job fails?
+
+Mattermost provides the status of each AD/LDAP sync job in **System Console** > **Authentication** > **AD/LDAP**. Here you can see the number of users updated and if the job succeeded or failed.
+
+Morever, any failures are returned in the server logs. The error log begins with the string ``Failed job`` and include a job_id key/value pair. AD/LDAP sync job failures are identified with worker name ``EnterpriseLdapSync``. You can optionally create a script that programmatically queries for such failures and notifies the appropriate system.
```
Comment (amyblais) [inline source/deployment/sso-ldap.md]:
```suggestion
Morever, any failures are returned in the server logs. The error log begins with the string ``Failed job`` and includes a job_id key/value pair. AD/LDAP sync job failures are identified with worker name ``EnterpriseLdapSync``. You can optionally create a script that programmatically queries for such failures and notifies the appropriate system.
```

**Pull request merged**